packages/common/package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@endo/common",
   "version": "1.0.1",
-  "private": true,
+  "private": false,
```

Comment:
The field can be omitted. Harmless if present and false.

Reply:
Done